static/docs/commands-reference/status.md

```diff
@@ -6,12 +6,15 @@ cache and local files, or between the local cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-q | -v] [-j JOBS] [--show-checksums]
-                  [-c] [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c] [-r REMOTE]
```

Comment:
why this change? no need to be precise and copy it exactly from the DVC command line output

Reply:
I did the same, simply copy-paste from command line output.

Reply:
_no_ need to copy it from the CLI output.

it makes sense to chage `[-q | -v]` but keep multiple lines to avoid scrolling. Especially on the mobile screen.